src/utils/bridgeMonitor.js

```diff
@@ -110,22 +157,39 @@ const inquiryAndUpdateDonationStatusFromBridge = async ({ app, donation }) => {
 
 const syncDonationsWithBridge = async app => {
   const donationService = app.service('donations');
+  const query = {
+    status: DonationStatus.PAID,
+    txHash: { $exists: true },
+    bridgeStatus: {
+      // $exists: true,
+      $nin: [
+        DonationBridgeStatus.PAID,
+        DonationBridgeStatus.CANCELLED,
+        DonationBridgeStatus.EXPIRED,
+      ],
+    },
+  };
   const donations = await donationService.find({
     paginate: false,
     query: {
       $limit: 100,
-      // 3 * 24 * 60 * 60 * 1000 means 3 days
-      createdAt: { $lte: new Date(Date.now() - 3 * 24 * 60 * 60 * 1000) },
-      status: DonationStatus.PAID,
-      txHash: { $exists: true },
-      bridgeStatus: {
-        // $exists: true,
-        $nin: [
-          DonationBridgeStatus.PAID,
-          DonationBridgeStatus.CANCELLED,
-          DonationBridgeStatus.EXPIRED,
-        ],
-      },
+      $or: [
+        {
+          ...query,
+          // 3 * 24 * 60 * 60 * 1000 means 3 days
+          createdAt: { $lte: new Date(Date.now() - 3 * 24 * 60 * 60 * 1000) },
```

Comment:
Do we still need this?

Reply:
@aminlatifi
I think we can get rid of this condition, 
if you agree I remove it

Reply:
It's not a big deal, let's keep it